gcovr/gcov_parser.py

```diff
@@ -53,7 +53,7 @@
 from .utils import Logger
 
 _EXCLUDE_LINE_FLAG = "_EXCL_"
-_EXCLUDE_LINE_PATTERN = re.compile(r"([GL]COVR?)_EXCL_(LINE|START|STOP)")
+_EXCLUDE_LINE_PATTERN = r"_EXCL_(LINE|START|STOP)"
```

Comment:
Can we add a `_POSTFIX` to the variable name?

Reply:
Sure thing. Will update in the morning (for me).

Reply:
Since I'm currently looking into gcovr as a replacement for the MS tool we are currently using, I would rather much like the possibility to configure not only the PREFIX but the WHOLE regexp.
E.g. our huge codebase (hundreds of thousands of lines)
has already code marked with exclusion markers and they look like this:
start of an excluded block: COVINFO BLOCK
end of a block: COVINFO BLOCKEND
exclude a line: COVINFO LINE
There is also a way to exclude a whole file, where the comment in the first line is: COVINFO FILE

Would be great if I would not need to change that in all files!

Reply:
If @Spacetown and others agree to that, I can update my PR to support. Otherwise, I will continue with the prefix path.

Reply:
@kollix At the moment I don't want to support this because we need three options to support this. The current used postfixes are also used by gcov and lcov. From which tool are your markers?

@zphensley42 Please continue with the prefix path.

Reply:
Will do.